### ISSUE TITLE
Clarify and document scope of RO

### ISSUE THREAD

**Issue by ramonawalls** [oborel/obo-relations#258]
Is RO intended to house ALL relations used in OBO ontologies? Different branches of RO have different levels of specificity, which may simply reflect demand. However, it seems that some domain ontologies (particularly those that share developers with RO) house all of their relations in RO, while others use their own ID space for domain specific relations, ideally as subclasses of RO relations. 

There are advantages and disadvantages to both of these methods, and I am not strongly committed to one approach or the other. However, it would be nice to have some community agreement about which approach to take, along with clear guidelines about how to create new relations under the chosen approach.

**Comment by cmungall** [comment]
Currently we have:

https://github.com/oborel/obo-relations/wiki/ROModules

Which does not fully answer your questions but is a start. We should also do a better job of annotating relations with subsets (we have done this for ecocore). 

We have also discussed this on some of the calls but I don't think any firm conclusions were reached.

**Comment by diatomsRcool** [comment]
What are some examples of relations in domain ontologies?

**Comment by jamesaoverton** [comment]
OBI has relations such as has_specified_input http://purl.obolibrary.org/obo/OBI_0000293 that relate planned processes to their inputs.

**Comment by mellybelly** [comment]
I suggest to have them all in RO to support more effective reuse, with some creative subset releases so that one doesn't have to import everything. (we could also do same with metadata/annotation props too)

**Comment by jamesaoverton** [comment]
@mellybelly I disagree. Domain ontologies should continue to define relations within their domains as needed. Putting all OBO relations in RO would mean putting all their domains and ranges in RO. Supporting effective reuse applies just as much to classes as relations, and the solution is not to put everything in RO. I think that the OBO Core we've begun discussing is a better approach.

**Comment by mellybelly** [comment]
Either way (subset extractions vs. implementation in individual ontologies) is fine with me, so long as there is some kind of coordination. Perhaps we could have some guidelines about how the individual ontology properties relate to those in RO would be enough? e.g. should they be subproperties? We just want to avoid duplication of content. A report of all properties for review on a regular cycle would be good too.

**Comment by nlharris** [comment]
Is this still needed? What are the action items here?
